### PR TITLE
No threadmarks drop down without javascript.

### LINKED ISSUE #19: Incorrect threadmarks menu behaviour with JS disabled
Disable javascript and hover over the threadmarks pagination button.

Threadmarks without JS

Woops!

### PR DESCRIPTION
**Pull request by sidane**
Fixes #19.

### CODE REVIEW

File: addon-sidaneThreadmarks.xml [hunk 1]
```diff
@@ -153,8 +153,12 @@
         </xen:if>
       </div>
     </div>
+    
+    <noscript>
+      <a href="{xen:link 'threads/threadmarks', $linkData}" rel="nofollow" class="threadmarksTrigger">{xen:phrase threadmarks}</a>
```
Comment (Xon) [inline addon-sidaneThreadmarks.xml:158]: Shouldn't this be "$threadmarks" to be consistant with the rest of this template?

Comment (Xon) [inline addon-sidaneThreadmarks.xml:158]: Wait I'm getting confused as this wants to link to all of the threadmarks, not just the recent threadmarks.

Comment (sidane) [inline addon-sidaneThreadmarks.xml:158]: Yeah that's the name of the variable being passed into the template by the pagination function.

File: addon-sidaneThreadmarks.xml [hunk 2]
```diff
@@ -153,8 +153,12 @@
...
+    </noscript>
   <xen:else/>
-    <a href="{xen:link 'threads/threadmarks', $threadmarks}" rel="nofollow" class="OverlayTrigger threadmarksTrigger">{xen:phrase threadmarks}</a>
+    <a href="{xen:link 'threads/threadmarks', $threadmarks}" rel="nofollow" class="OverlayTrigger threadmarksTrigger">{xen:phrase threadmarks}</a>
```
Comment (Xon) [inline addon-sidaneThreadmarks.xml]: This link is wrong, it should be using $linkData not $threadmarks, and probably should not have the class "OverlayTrigger"

Comment (sidane) [inline addon-sidaneThreadmarks.xml]: Good spot, fixed.

**Pull request merged**